eclipse-che/che-plugin-registry
# chore: remove che-theia, theia plugins and all references

### What does this PR do?
- Remove che-theia editor from the plugin registry
- Remove che-theia plugins from the registry
- Remove sidecars from the registry

![screenshot-eclipse-che apps ci-ln-smm7kc2-76ef8 aws-2 ci openshift [org-2023](https://issues.redhat.com//browse/org-2023) 03 28-17_21_22](https://user-images.githubusercontent.com/1271546/228352327-0df64316-987c-48c1-b894-94460cc0e105.png)

![screenshot-nimbusweb me-2023 03 29-16_44_28](https://user-images.githubusercontent.com/1271546/228558547-ae18afcd-5bff-45ad-ab4b-4f99d8daf7ef.png)

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/22090

### How to test this PR?

Check the plugin registry content generated by this PR: https://pr-check-1629-che-plugin-registry.surge.sh/ 
If you want to start a workspace:
- Use `quay.io/vsvydenk/che-plugin-registry:no-theia` image for plugin registry in CheCluster CRD
- workspace with che-code should start
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
